---
title: My curriculum vitae
author: Derick Hoskinson, Ph.D.
date: 2023-12-22
---

export const author = 'Derick Hoskinson, Ph.D.'

# {author}

[derickchoskinson@gmail.com](mailto:derickchoskinson@gmail.com)
[LinkedIn Profile](https://www.linkedin.com/in/dhoskinson)
[Personal Website](https://derickhoskinson.com)
[GitHub Profile](https://github.com/Derick80)

Phone: 312-871-8067

## Summary

I am a senior clinical scientist keen on using my expertise in variant curation and analysis to help patients and families. I am also a software developer focused on learning fundamental web technologies such as JavaScript, TypeScript, and React. I am open to new opportunities in the field of variant curation and analysis with a focus on developing web applications to work with genomic data.

## Work Experience

### Senior Clinical Scientist | Tempus AI

_Chicago, IL | March 2019 - Present_

- Created a tailored bed file to optimize genomic data analysis and facilitate the identification, classification, and clinical reporting of genetic variants on a large gene panel, resulting in improved accuracy and efficiency of downstream analysis.
- Contributed to the development and submission of the xT CDx assay to the FDA by curating and analyzing gene data to rank evidence per gene, ensuring accurate and effective clinical use
  -Played a vital role in the development and application of a new hereditary genetic testing panel, Tempus, xGv3 to identify germline variants in genes associated with hereditary cancer syndromes and incidental findings
- Identified gaps and deficiencies in current workflows and executed new workflows and SOPs to harmonize variant classification across multiple institutions
- Spearheaded the development and implementation of a reportable bed file solution for the Tempus xF assay, allowing for targeted reporting of specific genomic regions and improving the overall quality of data analysis.
- Used R programming language to design and implement a regions of interest bed file for reporting insertions and deletions for Tempus xF assay
- Used R programming language to create, update, and modify curated data to justify the inclusion or exclusion of genomic regions for xT-oncology FDA application
- Acted as the scientific lead in the development of the Tempus xF liquid biopsy panel to sequence and report SNVs and indels in clinically relevant regions of 105 genes plus CNVs and DNA rearrangements in a subset of these genes
- Used R programming language to programmatically access data, wrangle data, analyze data and present data to implement changes in workflow or other internal processes

### Variant Scientist | Tempus AI

_Chicago, IL | June 2017 - March 2019_

- Evaluated data and analyzed variants for a project that lead to the publication of the results in Nature BioTechnology
- Trained over 20 M.A.s, Ph.D.’s and post-baccalaureate in germline and somatic variant classification
- Worked closely with colleagues to build a team of variant scientists from 2 individuals to over 20 highly skilled variant scientists
- Gained proficiency in the R programming language and experience using Python, JavaScript, HTML, and CSS.

### Clinical Knowlege Curation Coordinator | Harvard Medical School, Partners Healthcare, Brigham and Women's Hospital

_Boston, MA | June 2015 - June 2017_

- Trained over 25 M.Ds, M.D./Ph.D.’s, rotating fellows, undergraduates, and post-baccalaureate in constitutional variant classification
- Assessed pathogenicity of variants from large gene panels covering cardiomyopathy, hearing loss, pulmonary disease, and Noonan syndrome
- Evaluated data and assessed variants as a critical member of the Geisinger MyCode project including the 56 genes recommended by the ACMG and 20 additional genes from Geisinger
- Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)
- Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD) Proficient in use of variant databases including- HGMD, ClinVar, LOVD, Deafness Variation database, Cardiodb, MitoMap, Leiden Muscular Dystrophy database, ARUP and COSMIC

### Medical Writer II | EBSCO Information Services, DynaMed

_Ipswich, MA | June 2014 - June 2015_

## Education

### Degree, University Name

_Location, Graduation Date_

- Details about your course of study.
- Any honors, awards, or special projects.

### Another Degree, Another University

_Location, Graduation Date_

- Details about your course of study or thesis.

## Skills

- Skill 1
- Skill 2
- Skill 3
- Any languages spoken and proficiency level.

## Certifications

- Certification Name, Issuing Organization – Date.
- Another Certification, Issuing Organization – Date.

## Publications

- [Title of Your Publication](link to publication), Publication Venue, Date.

## Conferences and Presentations

- "Title of Your Talk," Conference Name, Date.

## Professional Affiliations

- Organization Name, Role, Date Joined

## Awards and Honors

- Award Name, Awarding Organization, Date.

## Volunteer Experience

- Role, Organization, Date – Description of your role and contributions.

## Projects

### Project Name

- Brief description of the project.
- Technologies used, goals, and outcomes.
