export const work_experience = [
  {
    id: 1,
    institution: 'Tempus Labs',
    period: 'Mar 2019 - Present',
    title: 'Senior Clinical Scientist',
    duties: [
      {
        id: '1a',
        duty: 'Created a tailored bed file to optimize genomic data analysis and facilitate the identification, classification, and clinical reporting of genetic variants on a large gene panel, resulting in improved accuracy and efficiency of downstream analysis.'
      },
      {
        id: '1b',
        duty: 'Contributed to the development and submission of the xT CDx assay to the FDA by curating and analyzing gene data to rank evidence per gene, ensuring accurate and effective clinical use'
      },
      {
        id: 1,
        duty: 'Played a vital role in the development and application of a new hereditary genetic testing panel, Tempus, xGv3 to identify germline variants in genes associated with hereditary cancer syndromes and incidental findings'
      },
      {
        id: 2,
        duty: 'Identified gaps and deficiencies in current workflows and executed new workflows and SOPs to harmonize variant classification across multiple institutions'
      },
      {
        id: 3,
        duty: 'Spearheaded the development and implementation of a reportable bed file solution for the Tempus xF assay, allowing for targeted reporting of specific genomic regions and improving the overall quality of data analysis.'
      },
      {
        id: 4,
        duty: 'Used R programming language to design and implement a regions of interest bed file for reporting insertions and deletions for Tempus xF assay'
      },
      {
        id: 5,
        duty: 'Used R programming language to create, update, and modify curated data to justify the inclusion or exclusion of genomic regions for xT-oncology FDA application'
      },
      {
        id: 6,
        duty: 'Acted as the scientific lead in the development of the Tempus xF liquid biopsy panel to sequence and report SNVs and indels in clinically relevant regions of 105 genes plus CNVs and DNA rearrangements in a subset of these genes'
      },
      {
        id: 7,
        duty: 'Used R programming language to programmatically access data, wrangle data, analyze data and present data to implement changes in workflow or other internal processes'
      }
    ]
  },
  {
    id: 2,
    institution: 'Tempus Labs',
    period: 'Jun 2017 - Mar 2019',
    title: 'Clinical Scientist',
    duties: [
      {
        id: 1,
        duty: 'Evaluated data and analyzed variants for a project that lead to the publication of the results in Nature BioTechnology'
      },
      {
        id: 2,
        duty: 'Trained over 20 M.A.s, Ph.D.’s and post-baccalaureate in germline and somatic variant classification'
      },
      {
        id: 3,
        duty: 'Worked closely with colleagues to build a team of variant scientists from 2 individuals to over 20 highly skilled variant scientists'
      },
      {
        id: 4,
        duty: 'ained proficiency in the R programming language and experience using Python, JavaScript, HTML, and CSS. '
      }
    ]
  },
  {
    id: 3,
    institution: `Harvard Medical School, Partners Healthcare, Brigham and Women's Hospital`,
    period: 'Jun 2015 - Jun 2017',
    title: 'Clinical Knowlege Curation Coordinator',
    duties: [
      {
        id: 1,
        duty: 'Trained over 25 M.Ds, M.D./Ph.D.’s, rotating fellows, undergraduates, and post-baccalaureate in constitutional variant classification'
      },
      {
        id: 2,
        duty: 'Assessed pathogenicity of variants from large gene panels covering cardiomyopathy, hearing loss, pulmonary disease, and Noonan syndrome'
      },
      {
        id: 3,
        duty: 'Evaluated data and assessed variants as a critical member of the Geisinger MyCode project including the 56 genes recommended by the ACMG and 20 additional genes from Geisinger'
      },

      {
        id: 4,
        duty: 'Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)'
      },
      {
        id: 5,
        duty: `Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)
Proficient in use of variant databases including- HGMD, ClinVar, LOVD, Deafness Variation database, Cardiodb, MitoMap, Leiden Muscular Dystrophy database, ARUP and COSMIC`
      }
    ]
  },
  {
    id: 4,
    institution: `EBSCO Information Services, DynaMed`,
    period: 'Jun 2014 - Jun 2015',
    title: 'Medical Writer II',
    duties: [
      {
        id: 1,
        duty: `Wrote evidence based summaries of clinical trials aimed at serving as a resource for physicians and other health care professionals at point of care.`
      },
      {
        id: 2,
        duty: `Wrote evidence based point of care references for a number of clinically important disease topics`
      },
      {
        id: 3,
        duty: `Critically assessed published clinical trials for methodological integrity`
      }
    ]
  }
]
